### PR TITLE
Admin user index

### LINKED ISSUE #136: User Story 5, Admin Navigation
```
As an admin user
I see the same links as a registered user
Plus the following links
- a link to see all users ("/admin/users")

Minus the following links/info:
- a link to my shopping cart ("/cart")
- the count of the items in my cart
```

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [x] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #140

depends on #138

### PR DESCRIPTION
**Pull request by nicovigil1**
closes #136, adds new route for toggling user enable, adds view for admin user index

### CODE REVIEW

File: spec/features/users/admin/admin_user_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        @merchant = create(:user, role: 1)
+
+        visit root_path   
+        click_on 'Users'
+    end 
+
+    it "can see all default users" do
+        expect(page).to have_content(@user1.name)
+        expect(page).to have_content(@user2.name)
+        save_and_open_page
```
Comment (le3ah) [inline spec/features/users/admin/admin_user_index_spec.rb]: May want to remove save_and_open_page so we all don't hit it when we run RSpec; looks good though!

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -1,5 +1,16 @@
 class Admin::UsersController < Admin::BaseController
     def index
-        @users = User.all 
+        @default_users = User.default
```
Comment (tonyr729) [inline app/controllers/admin/users_controller.rb]: Don't need to change now but if this returns multiple default users I would like the method to be pluralized somehow maybe `default_users`
